nntrainer/src/activation_layer.cpp

```diff
@@ -53,18 +53,15 @@ int ActivationLayer::initialize(bool last) {
 }
 
 sharedConstTensor ActivationLayer::forwarding(sharedConstTensor in) {
-  input = in->clone();
+  input = *in;
   hidden = _act_fn(input);
```

Comment:
Can you please add a note for activation layer that the activation function is expected to work out of place.